openlibrary/templates/search/work_search_facets.html

```diff
@@ -16,15 +16,15 @@
         KeyError will be raised if key is not in the following dict.
         """
         facets = {
-            "has_fulltext": "Ebook",
-            "author_key": "Author",
-            "subject_facet": "Subjects",
-            "person_facet": "People",
-            "place_facet": "Places",
-            "time_facet": "Times",
-            "first_publish_year": "FirstPublished",
-            "publisher_facet": "Publisher",
-            "language": "Language",
+            "has_fulltext": _("Ebook"),
+            "author_key": _("Author"),
+            "subject_facet": _("Subjects"),
+            "person_facet": _("People"),
+            "place_facet": _("Places"),
+            "time_facet": _("Times"),
+            "first_publish_year": _("FirstPublished"),
+            "publisher_facet": _("Publisher"),
+            "language": _("Language"),
```

Comment:
We'll want to confirm that, in cases where we have dictionaries like this, that the values aren't being used as `keys` elsewhere in the code -- otherwise, once we i18n these keys, we could unintentionally break things. I don't think that's what's happening here, just sharing a possible use case we may hit given this PR touches some 100 files

Reply:
Roger that! I'll investigate -- and I believe this is the only dict affected by the PR (the script doesn't actually detect them, I just ran across it).